setup.py

```diff
@@ -23,10 +23,11 @@
 
 setup(
     name='fuel',
-    version=version,  # PEP 440 compliant
+    version='0.1.1',
```

Comment:
Why not use the `version` variable? This is done in order to avoid having the version defined in two separate places (`setup.py` and `fuel/__init__.py`).